examples/terraform/policy/gke.rego

```diff
@@ -1,12 +1,5 @@
 package main
 
-deny[sprintf("unable to find a gke container resource %s %s", [input.Resources[i].Type, input.Resources[i].Name])] {
-  not check_for_container_resources
-}
-
-check_for_container_resources {
-    some i
-    name := input.Resources[i].Name
-    type := input.Resources[i].Type
-    startswith(type, "google_container")
-}
+deny[sprintf("could not find any resources in: %v", [input])] {
```

Comment:
Any reason to move to a less specific example?

Reply:
While this policy is in the `examples` directory, it is actually being used as a test fixture. As such we are only providing enough verbosity to exercise the desired behavior under test.

It is not meant to be an example or reference file to show a user all the features and ways to leverage the terraform evaluation.

I feel this highlights that long term, we might consider moving our files used for acceptance tests into `testdata` and maintain a separate `examples` set of files to provide guidance and documentation to our users. 

Thoughts?

I also feel we can handle that in a later PR. 

Side note: this also calls out that this changes the object structure for the terraform evaluation, which should be considered a breaking change when versioning any release based off of this PR.